### ISSUE TITLE
Elements must only use allowed ARIA attributes

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1227]
Ensures ARIA attributes are allowed for an element's role
(https://dequeuniversity.com/rules/axe/4.7/aria-allowed-attr?application=AxeChrome)

Element Location:
```
#sbox-iframe i[aria-labelledby="cp-tab-contacts"]
<i class="fa fa-address-book" aria-labelledby="cp-tab-contacts"></i>
```
```
#sbox-iframe i[aria-labelledby="cp-tab-link"]
<i class="fa fa-link" aria-labelledby="cp-tab-link"></i>```
```

To solve this problem, you need to fix the following:
aria-labelledby attribute cannot be used on a i with no valid role attribute.

![Image](https://github.com/cryptpad/cryptpad/assets/95225431/4d6bf703-2912-4ed4-a25d-f03a4a10dfe7)

**Comment by davidbenque** [comment]
as far as I can tell the fix here is ont correct

On small screens only the icons are shown for non-selected tabs, hence the decision to use them as the item that is labelled. Otherwise they would have been aria-hidden

Now the tab title is aria-hidden and the icons are not but the readable label has been removed from the icon

**Comment by davidbenque** [comment]
>  aria-labelledby attribute cannot be used on a i with no valid role attribute.

it could be that the fix involves reverting https://github.com/cryptpad/cryptpad/commit/15727d605b3ec29494905393b8ef52670d50f76f and adding a role to the `i` elements instead (which role to use TBC)
